package.json

```diff
@@ -86,6 +87,7 @@
   "scripts": {
     "start": "node scripts/start.js",
     "build": "node scripts/build.js",
+    "convert": "node -r @std/esm scripts/convert.js < resources/cities.csv > public/data/smartCitiesData.json",
```

Comment:
Oh BTW - I already added this - if you rebase off master you'll see. Best to remove this line

Reply:
Fixed